### PR TITLE
[WIP] Fix error in DateFormat for ch.bav.sachplan-infrastruktur-schiene_anhorung

### PR DESCRIPTION
**Pull request by Sgachet**
Fix for #2757

### CODE REVIEW

File: chsdi/templates/htmlpopup/sis_planning.mako [hunk 1]
```diff
@@ -16,9 +16,9 @@ coordinationlevel_text = 'coordlevel_text_%s' % lang
 planningstatus_text = 'plstatus_text_%s' % lang
 description_text = 'description_%s' % lang
 dateto = '-'
-datefrom = datetime.datetime.strptime(c['attributes']['validfrom'].strip(), "%Y-%m-%d").strftime("%d.%m.%Y")
+datefrom = datetime.datetime.strptime(c['attributes']['validfrom'].strip(), "%Y-%m-%dT%H:%M:%S").strftime("%d.%m.%Y")
```
Comment (oterral) [inline chsdi/templates/htmlpopup/sis_planning.mako]: You could also  test if `validfrom` exists like you do for `validuntil` . Other wise there will be the same error fixed by @ltclm in https://github.com/geoadmin/mf-chsdi3/pull/2760/files

**Comment by ltclm** [comment]
https://github.com/geoadmin/mf-chsdi3/pull/2760

**Comment by AFoletti** [comment]
wait to merge this, I just fixed the data (deploy to prod today)

**Comment by AFoletti** [comment]
This PR is not needed. The FME now always write "%Y-%m-%d".
See https://github.com/geoadmin/bgdi-scripts/pull/245

The "try:" is handled in #2760

**Pull request closed without merging**